src/main/java/seedu/address/logic/commands/notes/NoteCommand.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.logic.commands.notes;
+
+import seedu.address.logic.commands.Command;
+
+/**
+ * Represents a note command with hidden internal logic and the ability to be executed.
+ */
+public abstract class NoteCommand extends Command {
+
+    public static final String COMMAND_WORD = "notes";
```

Comment:
Maybe "note" might be a better command word, since all these commands only change one note

Reply:
I think `notes` being the command word was meant to refer to the notes feature so `notes add` meant like going to the notes feature and adding a note. I've changed `notes` to `notebook` if that better clarifies the command?

Reply:
I think this risks making it more confusing.
The other commands are something like "detail add" or "question add" to tell people you are adding one detail/question. "notebook add" gives the message that you are adding a notebook, and "notes add" tells people you are adding multiple. I think "note" is still a better alternative.

Reply:
Okay thanks for pointing it out! I've just changed the note keyword to `note`!